### PR TITLE
FIX Removing unnessecary button stylesheet and fixing ellipsis in pagination

### PR DESCRIPTION
**Pull request by ScopeyNZ**

### CODE REVIEW

File: templates/Includes/Pagination.ss [hunk 1]
```diff
@@ -27,9 +27,11 @@
                             </li>
                         <% else %>
                             <li class="disabled page-item">
-                                <span>
-                                    <span aria-hidden="true">&hellip;</span>
-                                </span>
+                                <a href="#"
+                                   class="page-link"
```
Comment (robbieaverill) [inline templates/Includes/Pagination.ss]: Can you add this to the span so it's not detected as an anchor at all?

Comment (robbieaverill) [reply templates/Includes/Pagination.ss]: Or add `disabled` class, which removes pointer events

Comment (robbieaverill) [reply templates/Includes/Pagination.ss]: Actually you can do both - use a span and the disabled class, it looks right, disables pointer events and keyboard navigation

```html
<li class="page-item">
  <span class="page-link disabled" href="#">&hellip;</span>
</li>
```

Comment (ScopeyNZ) [reply templates/Includes/Pagination.ss]: Wow okay I thought I tried all variants of span and it wasn't working, but it's totally fine 😖

**Pull request merged**